cmd/root.go

```diff
@@ -101,6 +101,11 @@ func AddRepoDisambiguatorFlags(cmd *cobra.Command) {
 	_ = cmd.MarkFlagDirname("directory")
 }
 
+func AddOutputFormatFlag(cmd *cobra.Command) {
+	cmd.Flags().String("format", "plain", "output format")
```

Comment:
The flag help text could/should also list all possible format types (plain, json).
Like it is done e.g. in `repo_add.go` ->`repoAddCmd.Flags().StringP("type", "t", "", fmt.Sprintf("type of repo to add. One of [%s]", strings.Join(repos.SupportedTypes, ", ")))
`

Reply:
agreed